cookbooks/omnitruck/recipes/default.rb

```diff
@@ -15,14 +15,16 @@
   action :upgrade
 end
 
-hab_package 'chef-es/omnitruck-app'
-hab_package 'chef-es/omnitruck-poller'
-hab_package 'chef-es/omnitruck-web'
-hab_package 'chef-es/omnitruck-web-proxy'
+packages = %w(omnitruck-app omnitruck-poller omnitruck-web omnitruck-web-proxy)
+
+packages.each do |pkg|
+  hab_package "chef-es/#{pkg}" do
+    version node['applications'][pkg]
```

Comment:
@wrightp Nope just wanted to make sure it was being set in the `default/attributes.rb` so app version pinning promotion works as y'all expect (and `node['applications'][pkg]` resolves in all environments).